runtime/queries/wren/highlights.scm

```diff
@@ -0,0 +1,81 @@
+((identifier) @variable.builtin
+ (#match? @variable.builtin "^(Bool|Class|Fiber|Fn|List|Map|Null|Num|Object|Range|Sequence|String|System)$"))
+
+(call_expression
+  (identifier) @function)
+
+(method_definition
+  (identifier) @function.method)
+
+((parameter) @variable.parameter)
+
+(comment) @comment
+(string) @string
+(raw_string) @string
+(number) @constant.numeric.integer
+(identifier) @variable
+(ERROR) @error
```

Comment:
```suggestion
```

In the future we want to make ERROR highlighting an option in the language config. Since it's usually very noisy we disable it by default

Reply:
Removed.